setup.py

```diff
@@ -42,9 +42,11 @@
     include_package_data=True,
```

Comment:
The contrast is low for the code-formatted text in the light theme, which makes it a little hard to read, especially if it's used in a title (and thus shows up on the sidebar)

![Screenshot from 2021-07-16 12-22-52](https://user-images.githubusercontent.com/16181459/125978996-5b4d1be2-469f-4712-9496-59d6c23e8292.png)

Compare to dark theme:

![Screenshot from 2021-07-16 12-24-34](https://user-images.githubusercontent.com/16181459/125979091-9f3e3219-1f26-4d21-ada5-667aba8c6cb3.png)

Reply:
Agreed, that looks much better in the dark theme!

Reply:
Changed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3355/commits/3824ece8231dafcb7e21a98e81e7eabe001ab5c0